pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -935,22 +949,57 @@ func validateMatchCondition(v *admissionregistration.MatchCondition, opts valida
 	return allErrors
 }
 
-func validateValidation(v *admissionregistration.Validation, paramKind *admissionregistration.ParamKind, opts validationOptions, fldPath *field.Path) field.ErrorList {
+func validateVariable(compiler plugincel.Compiler, v *admissionregistration.Variable, paramKind *admissionregistration.ParamKind, opts validationOptions, fldPath *field.Path) field.ErrorList {
+	var allErrors field.ErrorList
+	trimmedName := strings.TrimSpace(v.Name)
```

Comment:
Anything else we should validate about the name? Should we restrict to valid CEL identifier strings (via a simple regex)?

Reply:
restricted to CEL identifier, according to the language spec.

Reply:
implemented. test added/